src/dotnet-uninstall/Windows/ProcessHandler.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Exceptions;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Windows
+{
+    public static class ProcessHandler
+    {
+        private const int UNINSTALL_TIMEOUT = 5 * 60 * 1000;
+
+        [DllImport("shell32.dll", SetLastError = true)]
+        static extern IntPtr CommandLineToArgvW(
+            [MarshalAs(UnmanagedType.LPWStr)] string lpCmdLine,
+            out int pNumArgs);
+
+        internal static void ExecuteUninstallCommand(IEnumerable<Bundle> bundles)
+        {
+            if (!IsAdmin())
+            {
+                RunAsAdmin();
+                return;
+            }
+
+            foreach (var bundle in bundles.ToList().AsReadOnly())
+            {
+                var args = ParseCommand(bundle.UninstallCommand, out var argc);
```

Comment:
what is "argc"

Reply:
`argc` is a commonly used variable for argument count, originated in C.

Reply:
not a fan of c, c++ naming style. but i can find it on the internet. so that is fine